packages/medusa/src/api/routes/admin/currencies/list-currencies.ts

```diff
@@ -21,6 +21,7 @@ import { FeatureFlagDecorators } from "../../../../utils/feature-flag-decorators
  *       type: boolean
  *       x-featureFlag: "tax_inclusive_pricing"
  *   - (query) order {string} A field to sort order the retrieved currencies by.
+ *   - (query) q {string} term used to search currencies' name and code.
```

Comment:
nit: 
```suggestion
 *   - (query) q {string} Term used to search currencies' name and code.
```
Otherwise I think the docs entry will look strange :D